Modules/ProfileManager/ProfileManager.lua

```diff
@@ -135,35 +151,28 @@ function ProfileManager:RemoveProfile(GUID)
     LOG:Debug("Remove profile: [%s]", GUID)
     LedgerManager:Submit(LEDGER_PROFILE.Remove:new(GUID), true)
 end
- -- TODO to do with the markings if profile is removed
-function ProfileManager:MarkAsAltByNames(main, alt)
+
+function ProfileManager:MarkAsAltByNames(alt, main)
```

Comment:
This function is not used (it is part of the GUI for alt linking). Therefore I do not think the code should be active in this PR.
Since it is never used it is not tested...
I propose just commenting out this whole function for now.

Reply:
this will be used